app/javascript/components/patient/Patient.js

```diff
@@ -206,8 +207,13 @@ class Patient extends React.Component {
               <Col className="text-truncate">
                 <span className="font-weight-normal">Phone:</span>{' '}
                 <span className="font-weight-light">
-                  {this.props.details.primary_telephone && `${this.formatPhoneNumber(this.props.details.primary_telephone)}`}
+                  {this.props.details.primary_telephone && `${this.formatPhoneNumber(this.props.details.primary_telephone)} `}
                 </span>
+                {this.props.details.blocked_sms && (
+                  <Form.Label className="nav-input-label text-danger">
```

Comment:
This has a margin below it that's causing there to be added spacing. Can you remove that margin? `m-0` to the `className` should work. It may also be because this is a Form.Label and could be resolved by changing to a `span` as Adam mentioned above.

![Screen Shot 2020-11-13 at 1 12 09 PM](https://user-images.githubusercontent.com/11698457/99106274-fe405480-25b1-11eb-8b4b-99e251f21506.png)